stylesss.css

```diff
@@ -3,10 +3,11 @@ h1 {
 }
```

Comment:
Some notations were changed as the Stylelint requirements. [see here](https://stylelint.io/user-guide/rules/color-function-notation/) for more details.